test/codegen/__snapshots__/codegen.js.snap

```diff
@@ -1476,6 +1476,91 @@ exports[`codegen #formats check we can convert all formats from namespace unvers
         },
         "country": {
           "type": "string"
+        },
+        "dictionary1": {
+          "$id": "org.acme.hr.Address.dictionary1",
+          "schema": {
+            "title": "Map1",
+            "description": "An instance of org.acme.hr.Map1",
+            "type": "object",
+            "propertyNames": {
+              "type": "string"
+            },
+            "additionalProperties": {
+              "type": "string"
+            }
+          }
+        },
+        "dictionary2": {
+          "$id": "org.acme.hr.Address.dictionary2",
+          "schema": {
+            "title": "Map2",
+            "description": "An instance of org.acme.hr.Map2",
+            "type": "object",
+            "propertyNames": {
+              "type": "string"
+            },
+            "additionalProperties": {
+              "type": "string"
+            }
+          }
+        },
+        "dictionary3": {
+          "$id": "org.acme.hr.Address.dictionary3",
+          "schema": {
+            "title": "Map3",
+            "description": "An instance of org.acme.hr.Map3",
+            "type": "object",
+            "propertyNames": {
+              "type": "string"
+            },
+            "additionalProperties": {
+              "type": "SSN"
+            }
+          }
+        },
+        "dictionary4": {
+          "$id": "org.acme.hr.Address.dictionary4",
+          "schema": {
+            "title": "Map4",
+            "description": "An instance of org.acme.hr.Map4",
+            "type": "object",
+            "propertyNames": {
+              "type": "string"
+            },
+            "additionalProperties": {
+              "type": "Concept"
+            }
+          }
+        },
+        "dictionary5": {
+          "$id": "org.acme.hr.Address.dictionary5",
+          "schema": {
+            "title": "Map5",
+            "description": "An instance of org.acme.hr.Map5",
+            "type": "object",
+            "propertyNames": {
+              "type": "string"
+            },
+            "additionalProperties": {
+              "type": "string"
+            }
+          }
+        },
+        "dictionary6": {
+          "$id": "org.acme.hr.Address.dictionary6",
+          "schema": {
+            "title": "Map6",
+            "description": "An instance of org.acme.hr.Map6",
+            "type": "object",
+            "propertyNames": {
+              "type": "string"
+            },
+            "additionalProperties": {
+              "type": "Employee",
```

Comment:
Is this line redundant? It may even be invalid.

I think that the following is sufficient
```
{ 
  "$ref": "#/definitions/org.acme.hr.Employee"
}
```

Reply:
Its redundant, and the validator will ignore redundant properties